diffusion/models/models.py

```diff
@@ -123,6 +123,122 @@ def stable_diffusion_2(
     return model
 
 
+def stable_diffusion_xl(
+    model_name: str = 'stabilityai/stable-diffusion-2-base',
```

Comment:
Do we want to default this to use the stable diffusion XL tokenizers and encoder(s), or does that not work until we have the full implementation?

Reply:
Waiting until the full implementation to do that, since there are two text encoders for SDXL.